src/components/domain/project/projectLinkModal/ProjectLinkModal.tsx

```diff
@@ -224,3 +300,24 @@ const LinkCompleteMessage = () => {
     />
   );
 };
+
+// 팀원 데이터 정렬 및 비회원/회원 구분선 index 구하기
+const processingMemberData = (memberData: MemberData[]) => {
+  // 회원가입이 안된 팀원 먼저 보여주기 - userId, 이름순 정렬
+  const sortedMemberData = memberData.sort((a, b) => {
```

Comment:
원본 배열을 직접 변경하는거보다 이게 더 안전하지 않을까요? 로직상 반드시 원본 배열을 직접 바꿔야 한다면 유지해도 됩니다
```suggestion
  const sortedMemberData = [...memberData].sort((a, b) => {
```

Reply:
✅ 8efb1c8eae8b12e10aeb992ca45d5353b7913a94